components/availability/src/Availability.svelte

```diff
@@ -408,7 +414,7 @@
         grid-template-columns: auto 1fr;
         gap: 0.5rem;
         height: 30px;
-        line-height: 30px;
+        line-height: 1.875rem;
```

Comment:
this can actually just be `1.875`, no rem needed :) `line-height` multiplies by the font-size